torchx/tracker/test/api_test.py

```diff
@@ -119,9 +120,18 @@ def test_env_factory_test(self) -> None:
             "torchx.tracker.api.load_group",
             return_value={"tracker1": tracker_factory},
         ):
-            app_run = tracker.app_run_from_env()
+            app_run = app_run_from_env()
             self.assertEqual(app_run.id, self.run_id)
-            self.assertEqual(TestTrackerBackend, type(app_run.backends[0]))
+            self.assertEqual(1, len(app_run.backends))
```

Comment:
likely can just change backends type to be Sized instead of Iterable

Reply:
yep just fixed.